### PR TITLE
169_majority_element.py

### PR DESCRIPTION
**Pull request by Rwatana**
Please check my code

### CODE REVIEW

File: 169_majority_element.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+class Solution:
+    def majorityElement(self, nums: List[int]) -> int:
+        nums.sort()
+        length = len(nums) // 2
+        if length == 0:
+            return nums[0]
+        for i in range(len(nums) - 1):
+            if nums[i] == nums[i + length]:
+                return nums[i]
```
Comment (ryujumoto) [inline 169_majority_element.py]: You can change this part so that the program is more efficient.

**Review by ryujumoto — COMMENTED** [review]
Done.

**Comment by Rwatana** [comment]
I change it to be effective.

**Comment by ryujumoto** [comment]
Refer [this code](https://github.com/okuda-seminar/leetcode_answers/blob/main/169_majority_element.py).

**Comment by Rwatana** [comment]
I understand that my way is ineffective and fixed the code.

**Review by ryujumoto — APPROVED** [review]

**Pull request merged**